### DISCUSSION TITLE
Installing own server, blank frontpage

### DISCUSSION THREAD

**Discussion by bostjanlaba** [mastodon/mastodon#21294] [Q&A] [answered]
Hi guys,

I've installed a server from scratch (dedicated, my own hardware). First I tried to setup Mastodon in a docker, didn't work. I ended up with blank front page (only Mastodon logo was shown). Then I reinstalled whole server with Ubuntu 22.04 LTS and as instructed here: 
https://docs.joinmastodon.org/admin/install/

I ended up the same. Please check https://lajna.si 

What am I missing?

Bostjan

**Comment by bostjanlaba** [comment, marked as answer]
Solved. mastodon user has to be part of www-data group and vice versa. I hope someone can add this to the above manuals.

**Reply by jdkruzr** [reply]
This has been needed since the last time I stood up a Mastodon instance in 2019 and it has never been added to the instructions. I have no idea why.

**Reply by hansenandrew** [reply]
Hi Guys, I'm having the same issue. Can you please let me know what commands you used to fix this issue. I've tried this sudo usermod -a -G www-data mastodon but didn't work. Thanks for your help

**Reply by naperwind** [reply]
Same issue for me. Added mastodon user to www-data and vice versa but still same issue.

**Comment by jdkruzr** [comment]
Try it in the other direction as well. That fixed it for me this time.

**Reply by hansenandrew** [reply]
Thank you for your reply, I tried it both directions but still not working. I will keep trying to fix it.

**Reply by bostjanlaba** [reply]
I can only say I also had to do it both ways, before it started working.
